addon/globalPlugins/remoteClient/local_machine.py

```diff
@@ -10,6 +10,8 @@
 import ctypes
 import braille
 import inputCore
+import systemUtils

```

Comment:
Currently NVDA Remote claims to be compatible with 2019.3 and later however `systemUtils` has been introduced in 2020.2 in https://github.com/nvaccess/nvda/pull/10493/. The following should ensure backwards compat:
```suggestion

try:
	from systemUtils import hasUiAccess
except ModuleNotFoundError:
	from config import hasUiAccess
	
```

Reply:
Great catch, thank you